public/src/admin/manage/category.js

```diff
@@ -64,7 +64,7 @@ define('admin/manage/category', [
 		}
 
 		
-		$('form.category input, form.category select')
+		$('form.category').find('input,textarea,select')
```

Comment:
Over time, I will try to push through my work in the main repository, as you asked me - small pieces.
https://github.com/NodeBB/NodeBB/pull/3602 ;)